sdk/core/Azure.Core.TestFramework/src/testproxy.json

```diff
@@ -0,0 +1,472 @@
+﻿{
+  "swagger": "2.0",
+  "info": {
+    "title": "Test Proxy",
```

Comment:
Do we manually update this file?

Reply:
Yes, but I can delete this as it was moved to tools repo.